bamboo/unit_tests/test_unit_layer_argmax.py

```diff
@@ -0,0 +1,232 @@
+import functools
+import operator
+import os
+import os.path
+import sys
+import numpy as np
+import pytest
+
+# Local files
+current_file = os.path.realpath(__file__)
+current_dir = os.path.dirname(current_file)
+sys.path.insert(0, os.path.join(os.path.dirname(current_dir), 'common_python'))
+import tools
+
+# ==============================================
+# Objects for Python data reader
+# ==============================================
+# Note: The Python data reader imports this file and calls the
+# functions below to ingest data. This is the only part of the script
+# that should be executed when the script is imported, or else the
```

Comment:
It looks like there quite a few lines in this file are run outside of functions (i.e. will run when imported).

Or do you mean only this part of some other script should be run on import here?

Reply:
Hm, looks like I need to word this better. My thinking is that the Python data reader only needs the stuff in this section when it imports this file, and it can ignore all of the model construction and PyTest stuff. In my original use-case, you would have two separate scripts to construct the model and to generate data, but we do this hacky approach so that everything conveniently fits in one script.